confidential/elgamal-registry-interface/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "spl-elgamal-registry-interface"
+version = "0.1.0"
+description = "SPL ElGamal Registry Interface"
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+bytemuck = { version = "1.23.1", features = ["derive"] }
+solana-instruction = "3.0.0"
+solana-program-error = "3.0.0"
+solana-pubkey = { version = "3.0.0", features = ["curve25519"] }
+solana-sdk-ids = "3.0.0"
+solana-zk-sdk = "4.0.0"
+spl-token-confidential-transfer-proof-extraction = { version = "0.5.0", path = "../proof-extraction" }
+
+[lib]
+crate-type = ["lib"]
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
```

Comment:
Just a small question. I see that the program id is included in the Cargo.toml for the token-2022 interface crate. Do we not need it here?